[DOCS] New "Getting Started" landing page (#1858)
elastic/security-docs: docs/getting-started/security-ui.asciidoc

[role="screenshot"]
image::images/collapse-side-nav-button.gif[width=100%][height=100%][Demo that shows how to to scroll through the page]

[float]
[[getting-started-page]]
=== Getting started page

> The Getting started page guides you to integrations that you can use to monitor your hosts and ingest data. Other {security-app} pages that show event data (for example, the Hosts and Network pages) display the getting started prompt until event data is available.

Reviewer: ```suggestion
The Getting started page guides you to integrations that you can use to monitor your hosts and ingest data. Other {security-app} pages that show event data (for example, the Hosts and Network pages) display the getting started prompt until event data has been ingested.
```
